### DISCUSSION TITLE
How to change Hololens 2 app icon?

### DISCUSSION THREAD

**Discussion by landnn** [microsoft/MixedRealityToolkit-Unity#9569] [Q&A] [answered]
By default the icon of each app deployed to the Hololens 2 is just a white square. I've tried to replace it by changing the "store logo" and the "default logo"  in the Unity player settings to my preferred image, but even after doing so the icon remains white. Is there another way to go about this?

**Comment by polar-kev** [comment]
Thanks for your question @landnn. @cre8ivepark will work on some documentation to explain this process.

**Comment by cre8ivepark** [comment, marked as answer]
Hi @landnn The best way to change the default app logo icon is using the Visual Studio's 'Visual Assets' Tab. Drag & Drop an icon and click 'Generate' button. Visual Studio will automatically generate all sizes.

![2021-03-31 10_59_08-MRTK Examples Hub - Microsoft Visual Studio](https://user-images.githubusercontent.com/13754172/113189858-84cb9a80-9210-11eb-8d50-fff7e9164dc2.png)
![2021-03-31 10_59_52-MRTK Examples Hub - Microsoft Visual Studio](https://user-images.githubusercontent.com/13754172/113189865-86955e00-9210-11eb-94f1-7aebaa1a1ba3.png)

For default Splash screen, you can change it in Unity's Player Settings > Splash Image
![2021-03-31 11_09_52-Unity 2018 4 31f1 -  PREVIEW PACKAGES IN USE  - PressableButtonExample unity - M](https://user-images.githubusercontent.com/13754172/113190840-a7aa7e80-9211-11eb-87da-b8f327800b51.png)

**Reply by landnn** [reply]
This is very helpful, thanks a lot!

**Comment by m-the-hoff** [comment]
Are you hoping to add a 2D icon or a 3D icon?  Adding an animated or non-animated 3D icon does require manual changes to the app manifest currently. A 2D icon does not.

**Reply by keveleigh** [reply]
>Adding an animated or non-animated 3D icon does require manual changes to the app manifest currently.

Not anymore! It's now built-in to the MRTK build settings, whether you're using the build window or Unity's regular build flow. We edit the manifest for you directly. https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9182

**Reply by m-the-hoff** [reply]
Woot!
